### PR TITLE
fix: usr: SDK-87: Fix code related to Nezha-Entities

### PR DESCRIPTION
**Pull request by adeshr**
Changed help messages wherever necessary, added a sub-parser for update code along with a few minor modifcations.

### CODE REVIEW

File: qds_sdk/nezha.py [hunk 1]
```diff
@@ -6,169 +6,251 @@
...
-    """
+    qds_sdk.NezhaDimension: is the class for retrieving and
+    manipulating Qubole's NezhaDimension.
 
-    """ endpoint for data source is  /nezha_data_sources endpoint"""
+    Endpoint for dimensions is: /nezha_cubes/{nezha_cube_id}/nezha_dimensions.
+    User needs to set it before using.
+    """
 
     rest_entity_path = ""
```
Comment (mindprince) [inline qds_sdk/nezha.py]: Why is this empty?

Comment (adeshr) [inline qds_sdk/nezha.py]: NezhaDimension/NezhaMeasure were needed in PyApi test and rest_entity_path needs NezhaCube Id. So, before using it there, we are setting rest_entity_path based on cube-id.
No command-line support is given for Dimensions/Measures.

**Comment by mindprince** [comment]
@vrajat Shouldn't this be called quark now?

File: qds_sdk/nezha.py [hunk 2]
```diff
@@ -6,169 +6,251 @@
...
+        parser_partition.add_argument(
+            "--table_name", dest="table_name", help="Table name")
+        parser_partition.add_argument(
+            "--destination_id", dest="destination_id", help="Data Source id")
+        parser_partition.add_argument("--query", dest="query", help="Query")
+
+        parser_ds = update_subparsers.add_parser(
+            "default_datasource", help="update default-datasource")
+        parser_ds.add_argument(
+            "default_datasource_id", help="Numeric id of the DataSource")
```
Comment (mindprince) [inline qds_sdk/nezha.py]: What does `update default_datasource ID` do? Does it change the default datasource to be a different ID? If that's the case, we should explain it in a better way.

**Pull request closed without merging**